hera_cal/abscal_funcs.py

```diff
@@ -1614,27 +1649,29 @@ def avg_data_across_red_bls(data, antpos, flags=None, broadcast_flags=True, medi
 
     # make red_data dictionary
     red_data = odict()
-    red_flags = odict()
+    red_wgts = odict()
 
     # iterate over reds
     for i, bl_group in enumerate(stripped_reds):
         # average redundant baseline group
         if median:
-            d = np.nanmedian(map(lambda k: data[k], bl_group), axis=0)
+            d = np.nanmedian(map(lambda k: data[k]*wgts[k], bl_group), axis=0)
+            d /= np.nanmedian(map(lambda k: wgts[k], bl_group), axis=0)
```

Comment:
Once again, this formula is incorrect for the median. See, e.g. https://en.wikipedia.org/wiki/Weighted_median

Reply:
I think I'll just get rid of this capability seeing as I actually have never even used it, such that we just stick to weighted means..